Dockerfile

```diff
@@ -72,8 +72,14 @@ RUN useradd -c $DEV_USER_NAME \
 		-s /bin/bash \
 		$DEV_USER
 
+# Add entrypoint to run gosu
+COPY entrypoint /
+ENTRYPOINT ["/entrypoint"]
+
 # Add default password for the SDK user (useful with sudo)
-RUN echo $DEV_USER:$DEV_USER_PASSWD | chpasswd
+# and replace entrypoint with dev user name
+RUN echo $DEV_USER:$DEV_USER_PASSWD | chpasswd && \
+	sed "s/\$DOCKER_USER/$DEV_USER/g" -i /entrypoint
```

Comment:
Can we set DOCKER_USER with @@DOCKER_USER@@ or similar to make it clear that we're really just replacing a string here?

Otherwise when I read just entrypoint it gives the impression that DOCKER_USER should be given as part of the environment.

Reply:
Fixed, is much easier to read with the @@DOCKER_USER@@